draft-cdn-control-header.md

```diff
@@ -75,11 +75,11 @@ shown here.
 
 # The CDN-Cache-Control Response Header Field
 
-The CDN-Cache-Control response header field allows origin servers to control the behaviour of gateway caches interposed between them and clients, separately from other caches that might handle the response.
+The CDN-Cache-Control response header field allows origin servers to control the behaviour of CDN caches interposed between them and clients, separately from other caches that might handle the response.
 
 It is a Dictionary Structured Header {{!I-D.ietf-httpbis-header-structure}}, whose members can be any directive registered in the HTTP Cache Directive Registry <https://www.iana.org/assignments/http-cache-directives/http-cache-directives.xhtml>.
 
-For gateway caches, when a valid CDN-Cache-Control is present in a response, it MUST take precedence over Cache-Control and Expires headers. In other words, CDN-Cache-Control disables all cache directives in other header fields, and is a wholly separate way to control the cache. Note that this is on a response-by-response basis; if CDN-Cache-Control is not present, caches MAY fall back to other control mechanisms as required by HTTP {{!I-D.ietf-httpbis-cache}}.
+When a valid CDN-Cache-Control is present in a response, CDN caches MUST ignore the Cache-Control and Expires response headers in that response. As such, CDN-Cache-Control is a wholly separate way to control the CDN cache. Note that this is on a response-by-response basis; if CDN-Cache-Control is not present, CDN caches MAY fall back to other control mechanisms as required by HTTP {{!I-D.ietf-httpbis-cache}}.
```

Comment:
Should that say "When a valid CDN-Cache-Control header is present"?

Reply:
This reminds me of an issue I want to bring up: do we want to support directives across multiple CDN-control-headers?

Reply:
```suggestion
When a valid CDN-Cache-Control header field is present in a response, CDN caches MUST ignore the Cache-Control and Expires response headers in that response. As such, CDN-Cache-Control is a wholly separate way to control the CDN cache. Note that this is on a response-by-response basis; if CDN-Cache-Control is not present, CDN caches MAY fall back to other control mechanisms as required by HTTP {{!I-D.ietf-httpbis-cache}}.
```

Reply:
@eaufavor it has to - an upstream intermediary might combine two headers (as with any header field, except Set-Cookie).